packages/rancher-gatekeeper/rancher-gatekeeper.patch

```diff
@@ -1012,18 +992,21 @@ diff -x '*.tgz' -x '*.lock' -uNr packages/rancher-gatekeeper/charts-original/val
  emitAuditEvents: false
  image:
 -  repository: openpolicyagent/gatekeeper
--  release: v3.1.0-rc.1
-+  repository: rancher/opa-gatekeeper
-+  tag: v3.1.0-rc.1
+-  release: v3.1.0
++  repository: rancher/openpolicyagent-gatekeeper
++  tag: v3.1.0
    pullPolicy: IfNotPresent
  nodeSelector: { kubernetes.io/os: linux }
  affinity: {}
-@@ -23,3 +23,8 @@
+@@ -23,3 +23,11 @@
    requests:
      cpu: 100m
      memory: 256Mi
 +global:
-+  systemDefaultRegistry: ""
++  cattle:
++    systemDefaultRegistry: ""
++    clusterId: ""
++    clusterName: ""
```

Comment:
are these being used in the chart? Otherwise imo they should be omitted since they don't actually configure anything. Users can always provide this if they want to even if it's not listed on the `values.yaml` (although it will do nothing).

Reply:
I'll remove them.